### PR TITLE
Add case with TextField

### PR DESCRIPTION
**Pull request by zarochintsev**
## Change summary
👋 Airbnb Team, I added case with text field, I would like to show how can we use `avoidsKeyboard` feature, I did not find it in examples.

### Screenshots
Main | Base | Keyboard
------- | ------- | -------
Screen Shot 2021-04-26 at 10 14 24 PM | Screen Shot 2021-04-26 at 10 14 26 PM | Screen Shot 2021-04-26 at 10 14 29 PM

## How was it tested?
*How did you verify that this change accomplished what you expected? Add more detail as needed.*
- [ ] Wrote automated tests
- [x] Built and ran on the iOS simulator
- [x] Built and ran on a device

## Pull request checklist
*All items in this checklist must be completed before a pull request will be reviewed.*

- [x] Risky changes have been put behind a feature flag, e.g. `CollectionViewConfiguration`
- [x] Added a [`CHANGELOG.md` entry](https://keepachangelog.com/en/1.0.0/) in the "Unreleased" section for any library changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/airbnb/epoxy-ios/compare/0.3.0...HEAD)
+### Added
```
Comment (erichoracek) [inline CHANGELOG.md:8]: nit: whitespace
```suggestion

### Added
```

File: Example/EpoxyExample/Data/Example.swift [hunk 2]
```diff
@@ -48,6 +51,8 @@ enum Example: CaseIterable {
       return "A CollectionView with a UICollectionViewFlowLayout"
     case .cardStack:
       return "A CollectionView with BarStackView items that have a card drawn around each stack"
+    case .textField:
+      return "An example that combines collections, bars, and text field"
```
Comment (erichoracek) [inline Example/EpoxyExample/Data/Example.swift:55]: nit: maybe include some details about keyboard avoidance?

File: Example/EpoxyExample/ViewControllers/TextFieldViewController.swift [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      dataID: DataID.username,
+      content: .init(
+        placeholder: "Username"),
+      style: .base)
+  }
+
+  private var buttonRowBar: BarModeling {
+    ButtonRow.barModel(
+      dataID: DataID.button,
+      content: .init(text: "Buy now"),
```
Comment (erichoracek) [inline Example/EpoxyExample/ViewControllers/TextFieldViewController.swift:46]: nit: maybe "submit" to better match the "username" field? I could imagine this demo would make sense as a fake login page if you'd want to add a fake password field too
```suggestion
      content: .init(text: "Submit"),
```

File: Example/EpoxyExample/ViewControllers/TextFieldViewController.swift [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  private lazy var bottomBarInstaller = BottomBarInstaller(
+    viewController: self,
+    avoidsKeyboard: true,
+    bars: [buttonRowBar])
+
+  private var textFieldRowItem: ItemModeling {
+    TextFieldRow.itemModel(
+      dataID: DataID.username,
+      content: .init(
+        placeholder: "Username"),
```
Comment (erichoracek) [inline Example/EpoxyExample/ViewControllers/TextFieldViewController.swift:39]:
```suggestion
      content: .init(placeholder: "Username"),
```

**Review by erichoracek — APPROVED** [review]
@zarochintsev Looks good to me! Just a few small nitpicks/suggestions

File: Example/EpoxyExample/Data/Example.swift (hunk 2 above)
Comment (zarochintsev) [reply Example/EpoxyExample/Data/Example.swift:55]: Sure, will do, thanks!

**Comment by zarochintsev** [comment]
@erichoracek this PR is ready for review again, thanks!

**Review by erichoracek — APPROVED** [review]

**Pull request merged**